docs/publishers/config.rst

```diff
@@ -385,6 +385,37 @@ loads.
   where annotations are located on the page relative to the current scroll
   position.
 
+.. option:: focus
+
+  .. warning::
+
+      This is an experimental API and may change in future.
+
+  ``Object``. A structured object that defines a focused filter set for the available 
```

Comment:
The period after `Object` looks peculiar to my eyes BUT is certainly the existing convention, so it's fine.